spacecmd/src/spacecmd/utils.py

```diff
@@ -51,7 +51,7 @@
 from tempfile import mkstemp
 from textwrap import wrap
 from subprocess import Popen, PIPE
-from dateutil.parser import parse as parse_datetime, ParserError
+from dateutil.parser import isoparse as parse_datetime, ParserError
```

Comment:
I can't see it from this diff in GitHub, is `ParseError` still used?

Reply:
It is not, I'll remove that from the import :+1: